app/src/main/assets/native/exoplayer.js

```diff
@@ -256,14 +256,44 @@ define(['events', 'appSettings', 'loading', 'playbackManager'], function (events
                             var audioCodec = codecs.audioCodecs[index];
                             audioCodecs.push(audioCodec.codec);
 
+                            var profiles = audioCodec.profiles.join('|');
+                            var maxChannels = audioCodec.maxChannels;
```

Comment:
No need to assign this to a variable first IMO. Can just use `audioCodec.maxChannels` in the 2 places that use it. Same for maxSampleRate.

Reply:
profiles can stay as-is since it manipulates the data (.join function)

Reply:
hm imo i prefer this approach, because if you need to change that name, you would need to change in 2 lines  (for this case) instead of just one. But if you insist, i can place in those 2 lines.

Reply:
I also think it's preferable, and it's only a small code style thing anyway.